xbmc/settings/lib/SettingsManager.cpp

```diff
@@ -162,30 +162,20 @@ bool CSettingsManager::Load(const TiXmlElement *root, bool &updated, bool trigge
   return true;
 }
 
-bool CSettingsManager::Save(TiXmlNode *root) const
+bool CSettingsManager::Save(const ISettingsValueSerializer* serializer, std::string& serializedValues) const
```

Comment:
> The `ColumnLimit` in `.clang-format` is set to `100` which defines line length [...]

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#3-formatting

Reply:
Fixed in #17062.